packages/svelte/test/preprocessors.test.ts

```diff
@@ -26,149 +27,251 @@ function expectComponentCodeToBeModified(
 }
 
 describe('componentTrackingPreprocessor', () => {
-  it('correctly inits the script preprocessor', () => {
-    const preProc = componentTrackingPreprocessor();
-    expect(preProc.script).toBeDefined();
-    expect(preProc.markup).toBeUndefined();
-    expect(preProc.style).toBeUndefined();
-  });
+  describe('script hook', () => {
```

Comment:
For reviewers: Nothing was changed in the tests within this new `describe` closure. I just added this level because the tests we had previously only test our preprocessor's `script` hook, while the new tests I added test the `markup` hook as well as the entire preprocessor